bootstrap.sh

```diff
@@ -22,5 +22,5 @@ else
 fi
 
 pip3 install -r $my_path/requirements.txt
-nohup python3 $my_path/app.py runserver &
+nohup python3 $my_path/app.py runserver > $my_path/web-application.log 2>&1 &
```

Comment:
The name of the log file is too long. How about - beyond.log?

Reply:
I think that "beyond.log" is not an indicative name, if we are using multiple components in our deployment (like DB, docker, web app, infrastructure tests and more) we need to have the log name file reflect the component, it will be faster and easier to find while debugging. 
I can agree with "web-app.log", but don't see much of a difference.